File: 01.Java-Basic/05.ConditionalStatementsAdvancedLab/src/InvalidNumber.java
import java.util.Scanner;

public class InvalidNumber {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int number = scanner.nextInt();
        boolean isValid = (number >= 100 && number <= 200) || number == 0;
        if (!isValid) {
            System.out.println("invalid");
        }
    }
}

File: JavaScript/01.JavaScriptBasic/05Nested-Conditional-Statements-Lab/src/InvalidNumber.js
function isValid(input) {
    let number = Number(input[0]);
    let valid = false;
    if ((number >= 100 && number <= 200) || number === 0) {
        valid = true;
    }

    if (!valid) {
        console.log("invalid");
    }
}

isValid(["75"]);
isValid(["150"]);
